spec/controllers/home_controller_spec.rb

```diff
@@ -74,8 +74,8 @@
 
       it 'filters out rentals not belonging to the current user' do
         get :index
-        expect(assigns[:rentals]).to include(create(:mock_rental, user_id: @user.id), create(:ongoing_rental, user_id: @user.id))
-        expect(assigns[:rentals]).not_to include(create(:mock_rental, user_id: @other_user.id))
+        expect(assigns[:rentals]).to include(create(:mock_rental, renter_id: @user.id), create(:ongoing_rental, renter_id: @user.id))
```

Comment:
Also, why does your factory have the word `mock` in it? Aren't they all mock things in a test?